INSTALL

```diff
@@ -9,6 +9,20 @@ wiki pages:
 * If you are in a hurry *
 *************************
 
+On Linux (Debian):
+
+ Note: you will need to add the Debian 'unstable' distribution to your source
+ file first.
+
+ Add the following line to your `/etc/apt/sources.list` file:
+
+ deb http://ftp.us.debian.org/debian unstable main contrib non-free
```

Comment:
technically this file is not a .md markdown file. I will try to incoporate your feedback.